rest/server_context.go

```diff
@@ -33,7 +33,7 @@ import (
 // The URL that stats will be reported to if deployment_id is set in the config
 const kStatsReportURL = "http://localhost:9999/stats"
 const kStatsReportInterval = time.Hour
-const kDefaultSlowServerCallWarningThreshold = 200 // ms
+const kDefaultSlowQueryWarningThreshold = 500 // ms
```

Comment:
Would this be dependent on the size of someone's data set?  Or have we set it 100X bigger than anything we expect to see under "normal" operating conditions?

Reply:
It's not 100x bigger - more like 10x than what we'd expect under heavy load.  (see https://hub.internal.couchbase.com/confluence/pages/viewpage.action?pageId=24579090)

It's configurable (similar to the previously existing view warning threshold), for customers that can live with high query latency, but it's targeted to a level that will provide log hints that view queries are much slower than we'd expect for a properly sized system.